x/records/migrations/v2/convert_test.go

```diff
@@ -121,7 +121,6 @@ func TestConvertUserRedemptionRecord(t *testing.T) {
 	}
 	expectedNewUserRedemptionRecord := recordstypes.UserRedemptionRecord{
 		Id:             id,
-		Sender:         sender,
```

Comment:
if you end up commenting out the field in `x/records/migrations/v2/convert.go` (nit, not required), should also comment this one out (for consistency)

Reply:
done